src/wkt/src/any.rs

```diff
@@ -222,6 +222,7 @@ impl crate::message::Message for Any {
 }
 
 /// Implement [`serde`](::serde) serialization for [Any].
+#[cfg_attr(not(feature = "_internal-semver"), doc(hidden))]
```

Comment:
It gave me pause to hide the `serde` traits from our messages, given that `serde` is a prereq for use with `wkt::Any`:

https://github.com/googleapis/google-cloud-rust/blob/74de4242705e73092a371371829f55eef2b1134a/src/wkt/src/any.rs#L172

... but I think the right fix is to enforce that our `Message`s are `serde`. I sent #2480

Reply:
Ack.